File: semaine5_imagier_nav/MiniNav_OC/MiniNav_OC/ViewController.m
//
//  ViewController.m
//  MiniNav_OC
//
//  Created by Cao Sang DOAN on 11/10/15.
//  Copyright (c) 2015 Cao Sang DOAN. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

MyView *myView;

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    UIScreen *screen = [UIScreen mainScreen];
    myView = [[MyView alloc] initWithFrame:screen.bounds];
    [self setView:myView];
    [myView release];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (void) viewWillTransitionToSize:(CGSize) size{
    [myView viewWillTransitionToSize:size];
}

- (BOOL)shouldAutorotate
{
    return YES;
}

- (BOOL) shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrientation
{
    return YES;
}

- (void) viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator
{
    [myView viewWillTransitionToSize:size];
}


@end

File: semaine7_location/iSouvenir/iSouvenir/ViewController.swift
//
//  ViewController.swift
//  iSouvenir
//
//  Created by Cao Sang DOAN on 18/11/15.
//  Copyright (c) 2015 Cao Sang DOAN. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    var myView: MyView?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        myView = MyView(frame: UIScreen.mainScreen().bounds)
        myView?.myController = self
        self.view = myView
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    override func viewWillTransitionToSize(size: CGSize, withTransitionCoordinator coordinator: UIViewControllerTransitionCoordinator) {
        myView!.positionner(size)
    }
    
}
